frontend/src/components/pages/Scheduling/VolunteerInformation.tsx

```diff
@@ -126,31 +131,34 @@ const VolunteerInformation = ({
 
   return (
     <Container variant="responsiveContainer">
+      <BackButton 
+        isBeingEdited={isBeingEdited}
+        onSaveClick={onSaveClick}
+        previous={previous}
+      />
       <SchedulingProgressBar activeStep={2} totalSteps={4} />
       <Text textStyle="mobileHeader2" mt="2em">
         Volunteer Information
       </Text>
       <Box
-        p="1.5em"
-        bg="squash.100"
+        p="2em"
+        bg="cottonCandy.100"
         align="left"
         m="2em 0 3em"
-        borderWidth="1px"
-        borderColor="#6C6C84"
         borderRadius="5px"
         maxWidth="500px"
       >
-        <Text textStyle="mobileHeader4">Proposed dropoff time</Text>
-        <Text textStyle="mobileBody">{new Date(startTime).toDateString()}</Text>
+        <Text textStyle="mobileHeader4">Proposed Drop-off Time</Text>
+        <Text textStyle="mobileBody">{new Date(startTime).toLocaleDateString([], {weekday: 'long', month: 'long', day: 'numeric' })}</Text>
         <Text textStyle="mobileBody">
-          {new Date(startTime).toLocaleTimeString()}-
-          {new Date(endTime).toLocaleTimeString()}
+          {new Date(startTime).toLocaleTimeString([], {hour: '2-digit', minute:'2-digit'}).replace(/^(?:00:)?0?/, '')}-
+          {new Date(endTime).toLocaleTimeString([], {hour: '2-digit', minute:'2-digit'}).replace(/^(?:00:)?0?/, '')}
         </Text>
```

Comment:
in the future, could consider using the `date-fns` library we have for formatting
https://date-fns.org/
i.e.  https://github.com/uwblueprint/community-fridge-kw/blob/a5b84214a2532c7d52aa7bd99ca59ce37705ee4f/frontend/src/components/pages/Dashboard/components/DropoffCard.tsx#L23